doc/related_projects.rst

```diff
@@ -297,11 +244,3 @@ Domain specific packages
 
 - `MSMBuilder <http://msmbuilder.org/>`_  Machine learning for protein
   conformational dynamics time series.
-
-- `scikit-surprise <https://surpriselib.com/>`_ A scikit for building and
-  evaluating recommender systems.
```

Comment:
There is already one link to Surprise Lib under "Recommender Systems"

Reply:
@NicolasHug sneaky sneaky /sarcastic